SpotifyArtistsViewer/SAArtist.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SAArtist.swift
+//  SpotifyArtistsViewer
+//
+//  Created by Kevin Zeckser on 6/6/16.
+//  Copyright © 2016 Kevin Zeckser. All rights reserved.
+//
+
+import UIKit
+
+struct SAArtist {
```

Comment:
No particular reason. Initially I thought I'd take advantage of the default initializer from structs.